io.openems.backend.metadata.odoo/src/io/openems/backend/metadata/odoo/EdgeCache.java

```diff
@@ -117,12 +115,12 @@ public synchronized MyEdge getEdgeForApikey(String apikey) {
 	}
 
 	/**
-	 * Gets all Edges as an unmodifiable Collection.
+	 * Returns a sequential stream with this {@link EdgeCache} as its source.
 	 *
-	 * @return a collection of Edges
+	 * @return a sequential {@link Stream} of the {@link MyEdge} in this cache.
 	 */
-	public Collection<Edge> getAllEdges() {
-		return Collections.unmodifiableCollection(this.edgeIdToEdge.values());
```

Comment:
What was the intention before to explicitely return a unmodifiableCollection? With this change we are loosing that safety measure.

Reply:
I had a look at the #2260 and i could only spot the following uses/differences in the MetadataOdoo class:

```java
@Override
	public Collection<Edge> getAllOfflineEdges() {
		// old: return this.edgeCache.getAllEdges().stream().filter(Edge::isOffline).toList();
		return this.edgeCache.stream().filter(Edge::isOffline) //
				.map(e -> (Edge) e).toList();
	}
```
@da-Kai was this change mainly for readability reasons? If so, I'd change this back to the original implementation.

Thank you in advance!